static/js/components/ThumbnailList.jsx

```diff
@@ -3,31 +3,71 @@ import PropTypes from 'prop-types';
 
 import dayjs from 'dayjs';
 import calendar from 'dayjs/plugin/calendar';
-import clsx from "clsx";
 
-import styles from "./ThumbnailList.css";
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
 
 dayjs.extend(calendar);
 
-const Thumbnail = ({ ra, dec, telescope, mjd, name, url }) => {
+
+/* const useStyles = makeStyles((theme) => ({
+})); */
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: 200,
+    margin: 'auto',
+    maxHeight: 500,
+    flexGrow: 1
+  },
+  title: {
+    fontSize: 14,
+  },
+  pos: {
+    marginBottom: 0,
+  },
+  mediaDiv: {
+    position: "relative"
+  },
+  media: {
+    height: 200,
+    width: 200,
+  },
+  crosshair: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    width: 200,
+    height: 200,
+    paddingBottom: "0.2em"
+  }
+}));
```

Comment:
Why not put this in `ThumnailList.css`?

Reply:
https://material-ui.com/styles/basics/#why-use-material-uis-styling-solution